bob/Cargo.toml

```diff
@@ -41,7 +41,7 @@ sysinfo = "0.26"
 tonic = { version = "0.6", features = ["prost"] }
 tower = "0.4"
 tower-service = "0.3"
-uuid = "1.1"
+uuid = "1.1.2"
```

Comment:
Why was the version restricted?

Reply:
Just in case

Reply:
I think it is better to revert and use `1.1`